docs_user/modules/proc_creating-a-ceph-nfs-cluster.adoc

```diff
@@ -105,7 +95,6 @@ ifeval::["{build}" != "upstream"]
 For more information on deploying the clustered Ceph NFS service, see the
 link:https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/7/html-single/operations_guide/index#management-of-nfs-ganesha-gateway-using-the-ceph-orchestrator[Management of NFS-Ganesha gateway using the Ceph Orchestrator (Limited Availability)] in _Red Hat Ceph Storage 7 Operations Guide_.
 endif::[]
```

Comment:
Is it okay to link to the Red Hat Ceph Storage 7 Operations Guide downstream? https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/7/html-single/operations_guide/index#management-of-nfs-ganesha-gateway-using-the-ceph-orchestrator

Reply:
Yes.
downstream we link the RHCS 7 ops guide, while upstream we point to docs.ceph.com.